.yamllint

```diff
@@ -0,0 +1,58 @@
+---
+
+yaml-files:
+  - '*.yaml'
+  - '.yamllint'
+
+rules:
+  anchors:
+    forbid-undeclared-aliases: true
+    forbid-duplicated-anchors: true
+    forbid-unused-anchors: true
+  braces:
+    forbid: false
+    min-spaces-inside: -1
+    max-spaces-inside: -1
+    min-spaces-inside-empty: -1
+    max-spaces-inside-empty: -1
```

Comment:
I would suggest:

```yaml
  braces:
    forbid: non-empty
```

Braces are not used in recipes. The only exception is `recipes/devel/cross-toolchain.yaml:73` where it's required.